index.html

```diff
@@ -614,6 +655,152 @@ <h4>
         </section>
       </section>
     </section>
+    <section>
+      <h2>
+        Automation
+      </h2>
+      <p>
+        The Device Posture API pose a challenge to test authors, as fully
+        exercising interface requires physical hardware devices. To address this
+        challenge this document defines a [[WEBDRIVER2]] [=extension commands=]
+        that allows users to control the reported device posture and simulate a
+        real device.
+      </p>
+      <h3>
+        Extension Commands
+      </h3>
+      <h4>
+        Set device posture
+      </h4>
+      <table class="simple">
+        <tr>
+          <th>
+            HTTP Method
+          </th>
+          <th>
+            [=extension command URI Template|URI Template=]
+          </th>
+        </tr>
+        <tr>
+          <td>
+            POST
+          </td>
+          <td>
+            /session/{session id}/deviceposture
+          </td>
+        </tr>
+      </table>
+      <p>
+        This [=extension command=] changes device posture to a specific
+        {{DevicePostureType}}.
+      </p>
+      <table class="simple">
+        <caption>
+          Properties of the parameters argument used by this algorithm
+        </caption>
+        <tr>
+          <th>
+            Parameter name
+          </th>
+          <th>
+            Value type
+          </th>
+          <th>
+          Required
+          </th>
+        </tr>
+        <tr>
+          <td>
+            posture
+          </td>
+          <td>
+            String
+          </td>
+          <td>
+            yes
+          </td>
+        </tr>
+      </table>
+      <p>
+        The [=remote end steps=] are:
+      </p>
+      <ol class="algorithm" data-cite="WEBDRIVER2">
+        <li>Let |posture| be the result of invoking
+        <a data-cite="!WEBDRIVER2#dfn-getting-properties">get a property</a>
+        "posture" from |parameters|.
+        </li>
+        <li>If |posture| is not a [=string=], return [=error=] with
+        [=error code|WebDriver error code=] [=invalid argument=].
+        </li>
+        <li>If |posture| is neither "{{DevicePostureType/continuous}}" nor
+        "{{DevicePostureType/folded}}", return [=error=] with
+        [=error code|WebDriver error code=] [=invalid argument=].
+        </li>
+        <li>Let |topLevelTraversable| be the
+        <a data-cite="webdriver2/#dfn-current-browsing-context">current browsing
+        context</a>'s [=browsing context/top-level traversable=].
+        </li>
+        <li>Set |topLevelTraversable|.<a data-link-for="top-level traversable">
+        [[\PostureOverride]]</a> to |posture|.
+        </li>
+        <li>Let |document| be |topLevelTraversable|'s [=navigable/active
+        document=]
```

Comment:
```suggestion
        document=].
```

Reply:
Fixed in https://github.com/JuhaVainio/W3C-device-posture/pull/1/commits/facde85c733cf1b19ad25a6eef6a281eed7e3fa9